carla/data/catalog/catalog.py

```diff
@@ -7,12 +7,25 @@
 
 
 class DataCatalog(Data):
-    def __init__(self, data_name, catalog_file):
+    def __init__(self, data_name, catalog_file, drop_first_encoding):
```

Comment:
Nice change! For the future, these kind of self-contained changes also make nice "mini-PRs" that get faster approvals

Reply:
👍